keyboards/s60_x/keymaps/ansi_qwertz/config.h

```diff
@@ -6,9 +6,8 @@
 // keymap needs oneshot functionality
 #undef NO_ACTION_ONESHOT
 
-#undef IS_COMMAND
 #define IS_COMMAND() ( \
```

Comment:
```suggestion
```
This is the same as the [existing definition](https://github.com/qmk/qmk_firmware/blob/master/keyboards/s60_x/config.h#L48-L50) for the keyboard, so this can be removed.